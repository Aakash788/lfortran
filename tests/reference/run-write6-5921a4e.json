{
    "basename": "run-write6-5921a4e",
    "cmd": "lfortran --no-color --run {infile} -o write6.f90.out",
    "infile": "tests/write6.f90",
    "infile_hash": "d1156f9891114f6be63e53f3abfc6b2bf8980e54727adbf8ea73eaa5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-write6-5921a4e.stdout",
    "stdout_hash": "53080fbd73388bd51a3aaaaf74ff6364f60547136f170eb8655751f1",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}