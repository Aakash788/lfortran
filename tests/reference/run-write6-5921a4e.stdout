12345
0002
-003
-0004
